brands/amenity/restaurant.json

```diff
@@ -940,6 +940,7 @@
     "tags": {
       "amenity": "restaurant",
       "brand": "IKEA Restaurant",
+      "cuisine": "swedish",
```

Comment:
I'm not sure this is entirely accurate. They serve a mix of all kinds of stuff. There's pizza, american breakfast, baked chicken at mine. I quick good search shows that some of them have philly cheese steaks, ribs, and fries. It's just whatever random selection of stuff they think will sell in the local market + meat balls.

Reply:
Then please remove this tag.

I bought some odd food items to go and I was under the impression that they sold little-known Swedish treats, and European typical foods. I haven't actually had lunch at one.

Reply:
I think it's ok.. We probably should put something in `cuisine` and that's as good as anything.   Yes there are hotdogs etc but they do kind of [play up the Swedish-ness of the menu](https://www.ikea.com/ms/en_KR/ikea_food/ikea_restaurant.html)..

More importantly, I'd switch it to `amenity=fast_food`, since it's a cafeteria where you serve and seat yourself.

Reply:
Just `amenity=fast_food` is ok.  I really don't want to add a bunch of extra tags that might not be true in all cases.